core/types/transaction_signing.go

```diff
@@ -72,7 +72,7 @@ func LatestSigner(config *params.ChainConfig) Signer {
 	if config.ChainID != nil {
 		switch {
 		case config.IsthmusTime != nil:
-			signer = NewIsthmusSigner(config.ChainID)
+			signer = NewIsthmusSigner(config.ChainID, config.IsOptimism() && config.Optimism.L2BlobTime != nil)
```

Comment:
Why do we use the same check function as above's `config.IsL2Blob()`

Reply:
I know the reason: we don't have the `blockNumber` and `blockTime `

Reply:
Yes